custom_components/tuya_local/devices/goldair_gpph900_heater.yaml

```diff
@@ -0,0 +1,172 @@
+name: Goldair Panel heater (GPPH770/GPPH780; GPPH900/GPPH910)
```

Comment:
This name should be generic (just "Panel heater"), branding belongs under products.